tests/Console/Commands/TweetImportantCFPDatesTest.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+use Carbon\Carbon;
+use Laracasts\TestDummy\Factory;
+use Mockery as m;
+use Symposium\Console\Commands\TweetImportantCFPDates;
+use Thujohn\Twitter\Twitter;
+
+class TweetImportantCFPDatesTest extends IntegrationTestCase
+{
+
+	public function setUp()
+	{
+		parent::setUp();
+
+		$this->twitter = m::mock(Twitter::class);
+	}
+
+	/** @test */
+	public function cfps_opening_or_closing_today_should_be_tweeted()
```

Comment:
This test is a bit confusing. Would you be up to write a few more to show why we're getting two tweets here? Maybe one test for opening and another for closing?

Thanks!